addresses/80001-mumbai-test.json

```diff
@@ -36,156 +45,159 @@
               "0x01ffc9a7"
             ]
           ]
         ],
         [
           "0x48e2b093",
           "0x1f931c1c",
-          "0x01ffc9a7"
+          "0x01ffc9a7",
+          "0x2ae6ea10"
         ]
-      ]
-    },
-    {
-      "name": "BuyerHandlerFacet",
-      "address": "0xe2439C0Bee3e3764a73113dd0b1E75408392166E",
-      "args": []
-    },
-    {
-      "name": "AgentHandlerFacet",
-      "address": "0xfae9F984E8F632634e6e1d979139770537Bca4Fe",
-      "args": []
+      ],
+      "interfaceId": ""
     },
     {
-      "name": "FundsHandlerFacet",
-      "address": "0x5980CE2C787B9033Ff578c4A60676d35C09887a6",
-      "args": []
+      "name": "AccountHandlerFacet",
+      "address": "0x03e583A035aFFF33ec0918a4c690BC9883C423a0",
+      "args": [],
+      "interfaceId": "0x15335ed7"
     },
     {
-      "name": "PauseHandlerFacet",
-      "address": "0x8528d132331Ec469A2d7CE412A7089Feb54392EC",
-      "args": []
+      "name": "SellerHandlerFacet",
+      "address": "0x419c208f9F62f43278190b58da59200f06e64619",
+      "args": [],
+      "interfaceId": "0x15335ed7"
     },
     {
-      "name": "BosonVoucher Beacon",
-      "address": "0xc3552C2F8331Be2fFAA1e80504A0813C3d8816C5",
-      "args": [
-        "0x785a225EBAC1b600cA3170C6c7fA3488A203Fc21",
-        "0x12b2194bdBDabE12B723c0865714327f940eFb63"
-      ]
+      "name": "BuyerHandlerFacet",
+      "address": "0x720a33545DbdcCe84666997eF7C4AfB6CC53E87e",
+      "args": [],
+      "interfaceId": "0x15335ed7"
     },
     {
-      "name": "BosonVoucher Proxy",
-      "address": "0xaC991233dB3EDAB7f5605ceb111738D5dd7B8484",
-      "args": []
+      "name": "DisputeResolverHandlerFacet",
+      "address": "0xE75dB950E786808d7606090e72E9770fA97Bd9D2",
+      "args": [],
+      "interfaceId": "0x15335ed7"
     },
     {
-      "name": "ERC165Facet",
-      "address": "0x8A8c38674005423fD4B102964fE5E05365a28F42",
+      "name": "AgentHandlerFacet",
+      "address": "0xcde27501927cFa468aAc6e41b468c25755189566",
       "args": [],
-      "interfaceId": "0x2ae6ea10"
+      "interfaceId": "0x15335ed7"
     },
     {
-      "name": "AccountHandlerFacet",
-      "address": "0x04e9635d14dE63Fb50f3e629d24F8153FE6629FA",
+      "name": "BundleHandlerFacet",
+      "address": "0xf45f4F696B2F330d41e43B1322E7CaFB1DEDDe7b",
       "args": [],
-      "interfaceId": "0x1f891681"
+      "interfaceId": "0x7b53dece"
     },
     {
-      "name": "ProtocolInitializationHandlerFacet",
-      "address": "0xAA7D28Bcc8127E706B0630208D53652F8BA306BD",
+      "name": "DisputeHandlerFacet",
+      "address": "0xD16606Ae160e348EEB50e71F1b8a2D891B14e7d3",
       "args": [],
-      "interfaceId": "0x0d8e6e2c"
+      "interfaceId": "0xd9ea8317"
     },
     {
-      "name": "OrchestrationHandlerFacet2",
-      "address": "0x6286d3d8E5ceCB5F17f4255957A5926450d37C21",
-      "args": []
+      "name": "ExchangeHandlerFacet",
+      "address": "0x9AE3F5C157656C4e86b5F8340F399Ee33C62792e",
+      "args": [],
```

Comment:
`args` should not be empty for facets with a constructor. That's important for contract verification.

Probably the deployment script needs to be changed to correctly store it.

Reply:
what needs to happen here, @zajck ? Do we need to create an issue so that we don't forget, or do we want it to be fixed as a part of this PR ?

Reply:
The minimum we need to change is to manually update this file and update it with the value that was used during the deployment (1).

We can update the deploy script in another PR, but it must be added to `v2.2.1` release. 
And I believe that the existing `v2.2.1` release should be changed to `v2.2.1-rc.1` and changed to pre-release, since there are some details missing to be called a release.

Reply:
done